src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseworkerSendOrder.java

```diff
@@ -115,20 +114,16 @@ public AboutToStartOrSubmitResponse<CaseData, State> aboutToSubmit(
             && caseData.getCicCase().getOrderIssuingType().equals(OrderIssuingType.ISSUE_AND_SEND_AN_EXISTING_DRAFT)) {
 
             selectedDynamicDraft = caseData.getCicCase().getDraftOrderDynamicList().getValue().getLabel();
-            String[] selectedDraft = selectedDynamicDraft.split(DOUBLE_HYPHEN);
 
-            for (ListValue<DraftOrderCIC> draftOrderCICListValue : caseData.getCicCase().getDraftOrderCICList()) {
+            for (ListValue<DraftOrderCIC> draftOrderCICValue : caseData.getCicCase().getDraftOrderCICList()) {
 
-                String[] draftOrderFile = draftOrderCICListValue.getValue()
-                    .getTemplateGeneratedDocument().getFilename().split(DOUBLE_HYPHEN);
-
-                if (selectedDynamicDraft
-                    .contains(draftOrderCICListValue.getValue().getDraftOrderContentCIC().getOrderTemplate().getLabel())
-                    && selectedDraft[1].equals(draftOrderFile[2])
-                ) {
-                    selectedDraftOrder = draftOrderCICListValue.getValue();
+                if (selectedDynamicDraft.contains(draftOrderCICValue.getValue().getDraftOrderContentCIC().getOrderTemplate().getLabel())) {
```

Comment:
@tomxelliott - It would be good to have some tests added/updated for these changes.

Reply:
@rajaram-nagarajan - I will revert the tests back to their state before the change that introduced the issue (https://github.com/hmcts/sptribs-case-api/pull/1273/files)

Reply:
Will add a test case

Reply:
Have reverted the dynamic list function so tests are reverted to previous if condition state from https://github.com/hmcts/sptribs-case-api/pull/1273/files#diff-b5f1ff28a4f4bef77a7943b958de494db3010e9fd7bb37323e7b229049e58702